cfn-resources/cluster/test/cfn-test-delete-inputs.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# cfn-test-delete-inputs.sh
+#
+# This tool deletes the mongodb resources used for `cfn test` as inputs.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage {
+	echo "usage:$0 "
+}
+
+projectId=$(jq -r '.ProjectId' ./inputs/inputs_1_create.json)
+
+#delete project
+if atlas projects delete "$projectId" --force; then
+	echo "$projectId project deletion OK"
+else
+	(echo "Failed cleaning project:$projectId" && exit 1)
+fi
```

Comment:
this will fail if there are clusters in the project

Reply:
@andreaangiolillo  Since we create project freshly everytime we run tests there is no chance to have clusters. And cfn test creates and deletes the cluster automatically as part of its testing.